src/utils/index.ts

```diff
@@ -0,0 +1,8 @@
+const goToOms = (token: string, oms: string) => {
+  const link = (oms.startsWith('http') ? oms.replace('api/', "") : `https://${oms}.hotwax.io/`) + `?token=${token}`
+  window.open(link)
```

Comment:
In the current implementation, we open the OMS in a blank window, so I think we should follow the same here.

Reply:
Updated as per the current implementation.